src/picongpu/include/plugins/kernel/CopySpecies.kernel

```diff
@@ -71,6 +71,7 @@ struct ConcatListOfFrames
              ++linearBlockIdx
              )
         {
+            T_Filter filter = filterWindowDomain;
```

Comment:
everything is by default shared in OpenMP if not `default(none)` is used.

@erikzenker yes I will add the comment